### ISSUE TITLE
Bootloop

### ISSUE THREAD

**Issue by CipherBKT** [EmperorEye1993/GPU-Turbo-Boost#9]
Installed the latest 6.2 version on my Mi A3 and it caused my device to bootloop...

**Comment by etzKarim** [comment]
same here on redmi note 8, the latest version caused my device to bootloop

**Comment by CipherBKT** [comment]
> same here on redmi note 8, the latest version caused my device to bootloop

Yeah, it's easy to fix using adb, just have to clear all magisk modules

**Comment by cbnn2405** [comment]
Installed the latest 6.2 version on my k20 pro ,miui 21.8.25 and it caused my device to bootloop

**Comment by NitroVenom** [comment]
imagine bootlooping because of a random build.poopoo tweaks mod that sets governor to performance without validating what the performance governor actually does
